recipes/napari-trackpy/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "napari-trackpy" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-trackpy-{{ version }}.tar.gz
+  sha256: 0f4097cbd1ef5fa616285c4b52e462896a1ec994b119576153ca4ca89e9d6416
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
```

Comment:
Could we sort this so it is easier to update and read :) ?

    - pip
    - python >=3.8
    - setuptools >=42.0.0
    - setuptools-scm
    - wheel

Reply:
Done!